src/Serilog.Sinks.Elasticsearch/LoggerConfigurationElasticSearchExtensions.cs

```diff
@@ -180,7 +181,8 @@ public static class LoggerConfigurationElasticsearchExtensions
             ILogEventSink failureSink = null,
             long? singleEventSizePostingLimit = null,
             int? bufferFileCountLimit = null,
-            Dictionary<string,string> templateCustomSettings = null)
+            Dictionary<string,string> templateCustomSettings = null,
+            ElasticOpType batchAction = ElasticOpType.Index)
```

Comment:
Since this is a binary breaking change, we need to clearly indicate this change in the changes.md